pkg/apis/kops/instancegroup.go

```diff
@@ -89,6 +90,9 @@ type InstanceGroupSpec struct {
 
 	// NodeLabels indicates the kubernetes labels for nodes in this group
 	NodeLabels map[string]string `json:"nodeLabels,omitempty"`
+
+	// Taints indicates the kubernetes taints for nodes in this group
+	Taints []string `json:"taints,omitempty"`
```

Comment:
The API does use []api.Taint, but I think a []string is much simpler, so I think we let this stand